lib/tasks/db_maintenance.rake

```diff
@@ -234,8 +234,7 @@ namespace :cartodb do
         count = ::User.where(database_host: database_host).count
       end
       execute_on_users_with_index(task_name, Proc.new { |user, i|
-        begin
-          log(sprintf("Trying on %-#{20}s %-#{20}s (%-#{4}s/%-#{4}s)...", user.username, user.database_name, i+1, count), task_name, database_host)
+        begin log(sprintf("Trying on %-#{20}s %-#{20}s (%-#{4}s/%-#{4}s)...", user.username, user.database_name, i+1, count), task_name, database_host) # Only load cartodb extensions once per database, i.e. for dedicated users and org owners
```

Comment:
Looks like this was just a formatting change, was this intentional?

Reply:
No, this should be reverted (though it doesn't affect functionality)